components/prize_pool/wikis/trackmania/prize_pool_custom.lua

```diff
@@ -43,6 +47,18 @@ function CustomLpdbInjector:adjust(lpdbData, placement, opponent)
 		Variables.varDefault('tournament_type')
 	)
 
+	local worldTourPoints = Array.filter(placement.parent.prizes, function (prize)
+		if prize.type == PRIZE_TYPE_POINTS and prize.data.title == PRIZE_TITLE_WORLD_TOUR then
+			return true
+		end
+	end)[1]
+
+	if worldTourPoints then
+		CustomPrizePool.addWTDatapoint(lpdbData, placement:getPrizeRewardForOpponent(opponent, worldTourPoints.id))
+	end
+
```

Comment:
```suggestion

	if worldTourPoints then
		lpdbData.extradata.prizepoints = placement:getPrizeRewardForOpponent(opponent, worldTourPoints.id)
		lpdbData.extradata.prizepointsTitle = 'wt_points'
	end

```

Reply:
Solution tested on the page linked in the first message, you can see LPDB extradata working as intended

Reply:
Do note that TeamCard will override extradata when in mainspace

Reply:
Should I use a custom TeamCard module ? Or is there any workaround ?

Reply:
I'm not sure whether trackmania is on the current commons TC, but if that's the case you should be able to add `Module:TeamCard/Custom` with a function that extends lpdb, like it is done on rainbowsix for example:
https://liquipedia.net/rainbowsix/Module:TeamCard/Custom